autoscheduler/frontend/src/types/Section.ts

```diff
@@ -1,6 +1,19 @@
 import Instructor from './Instructor';
 import Grades from './Grades';
 
+// Instructional methods from the backend, make sure these are synced with Django's Section model
```

Comment:
ya know this is the one of the down sides with having our backend and frontend written in separate languages. It would've been nice to share the same type files across backend and frontend just so we know we're always synced (also not switching between camel case and snake case would be nice)